package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yarn",
-  "version": "0.16.0",
+  "version": "0.16.2",
```

Comment:
I wonder, should we?
I was thinking about this earlier.

Our current release branching mechanism involves creating a branch from master.
After that, release branch and master live on their own.
We may publish patches to release branch by incrementing patch version but there is no point in linking release branch's patch to master because they are already most likely diverged.

There is still something wrong with master branch being 0.16.0 until 0.17.0 is cut.
Maybe we should address this to indicate that 0.16.0 on master is actually not in sync with release branch.